src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
+  --color: #00acdc;
```

Comment:
Create a variable with root selector

Reply:
Move all variables to the :root selector

Reply:
Still not fixed